Resources/Prototypes/ai_factions.yml

```
@@ -192,3 +192,14 @@
     - Xeno
     - Zombie
     - Dragon
+
+- type: npcFaction
+  id: SAN
+  hostile:
+  - NanoTrasen
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)